ibm/service/power/ibm_pi_constants.go

```diff
@@ -29,7 +29,13 @@ const (
 	Arg_VolumeGroupID                       = "pi_volume_group_id"
 	Arg_VolumeID                            = "pi_volume_id"
 	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
-
+	Arg_VPNPolicyDhGroup                    = "Arg_PolicyDhGroup"
+	Arg_VPNPolicyAuthentication             = "pi_policy_authentication"
+	Arg_VPNPolicyEncryption                 = "pi_policy_encryption"
+	Arg_VPNPolicyKeyLifetime                = "pi_policy_key_lifetime"
+	Arg_VPNPolicyName                       = "pi_policy_name"
+	Arg_VPNPolicyPresharedKey               = "pi_policy_preshared_key"
+	Arg_VPNPolicyVersion                    = "pi_policy_version"
```

Comment:
No need for vpn in the name.
```suggestion
	Arg_PolicyDhGroup                    = "Arg_PolicyDhGroup"
	Arg_PolicyAuthentication             = "pi_policy_authentication"
	Arg_PolicyEncryption                 = "pi_policy_encryption"
	Arg_PolicyKeyLifetime                = "pi_policy_key_lifetime"
	Arg_PolicyName                       = "pi_policy_name"
	Arg_PolicyPresharedKey               = "pi_policy_preshared_key"
	Arg_PolicyVersion                    = "pi_policy_version"
```